composer.json

```diff
@@ -12,13 +12,13 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "illuminate/notifications": "~6.0 || ~7.0 || ~8.0",
-        "illuminate/support": "~6.0 || ~7.0 || ~8.0"
+        "php": ">=8.1",
+        "illuminate/notifications": "~10.0",
```

Comment:
May as well add 11 here as it will be out soon.

Reply:
Done!